app/routes/events/view/speakers/create.js

```diff
@@ -1,4 +1,6 @@
 import Route from '@ember/routing/route';
+import { countries } from 'open-event-frontend/utils/dictionary/demography';
```

Comment:
Please do this import inside the form component file itself, instead of returning it in the model, as the countries are in a locally stored static dictionary.

Reply:
@CosmicCoder96 I've made the required changes. Please review :)